sbt/src/sbt-test/project/old-ops/test

```diff
@@ -1,17 +1,18 @@
 $ copy-file changes/settingAssign/build.sbt build.sbt
--> compile
+# Necessary to reload since the test assumes that the new build is picked up
+-> reload
 
 $ copy-file changes/settingAppend1/build.sbt build.sbt
--> compile
+-> reload
 
 $ copy-file changes/settingAppendN/build.sbt build.sbt
--> compile
+-> reload
 
 $ copy-file changes/taskAssign/build.sbt build.sbt
--> compile
+-> reload
 
 $ copy-file changes/taskAppend1/build.sbt build.sbt
--> compile
+-> reload
 
 $ copy-file changes/taskAppendN/build.sbt build.sbt
--> compile
+-> reload
```

Comment:
What I meant is that `sbt` was initialized lazily, only when `>` statements were invoked. `compile` here implied initializing sbt.